Sources/kai/AST.swift

```diff
@@ -76,6 +76,8 @@ indirect enum AstNode {
     case stmtIf(cond: AstNode, body: AstNode, AstNode?, SourceRange)
     case stmtReturn([AstNode], SourceRange)
     case stmtFor(initializer: AstNode?, cond: AstNode?, post: AstNode?, body: AstNode, SourceRange)
+    case stmtSwitch(cond: AstNode?, body: AstNode, defaultBody: AstNode?, SourceRange)
```

Comment:
Couple things:
- avoid having `AstNode.list` in the final AST as you can see above on `stmtReturn`, best to be consistent
- switch can be on more than just a condition (ie: you could switch on a type)
```swift
stmtSwitch(subject: AstNode?, cases: [AstNode], defaultCase: AstNode?, SourceRange)
```